packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -960,7 +960,7 @@ describe("normalizeAndValidateConfig()", () => {
 				first_party_worker: true,
 				logpush: true,
 				placement: {
-					mode: "smart",
+					mode: "off",
```

Comment:
So you had to change this to prevent the new validation error.
But now we don't have a check that placement config is not just defaulting to `off`, which is a shame.
Perhaps you could add an expectation check to one of the new tests below?

Reply:
Added this check to the `should not error if placement is configured and triggers is empty array` test!